docs/software-and-images-guidelines.md

```diff
@@ -0,0 +1,36 @@
+# Software and image guidelines
+
+## Software preinstallation policy
+In general, these are the guidelines we consider when deciding what to pre-install:
+
+- Tools and ecosystems that are broadly popular and widely-used will be given priority.
+- Recent versions of tools will be given priority over older versions.
+- Tools and versions that are deprecated or have reached end-of-life will not be added.
+- If a tool can be installed during the build, we will evaluate how much time is saved
+ and how much space is used by having the tool pre-installed.
+- MIT, Apache, and GNU licenses are ok, anything else we'll have to check with lawyers.
+- If a tool takes much space we will evaluate space usage and provide a decision if this tool can be pre-installed.
+- If a tool requires the support of more than one version, we will consider the cost of this maintenance, how often new versions bring dangerous updates.
```

Comment:
2 general nitpicks:
1) Some lines have `.` at the end. Some don't have. Let's do them consistent.
2) May be rewrite sentences that started with `If` to be consistent with other ones.

Reply:
I think, that some statements will look weird with the "If" in the beginning, so I'd leave this section as is.